index.js

```diff
@@ -37,9 +37,18 @@ function installXgboost(artifacts_version, xgboost_version) {
   executeCommand("sudo ldconfig", "fail to ldconfig");
 }
 
-function installLibbpf() {
-  core.info(`libbpf`);
-  executeCommand("sudo apt-get install -y libbpf-dev", "fail to install libbpf related package");
+function installLibbpf(libbpf_version) {
+  core.info(`installing libbpf version ` + libbpf_version + ` from source`);
+  executeCommand("mkdir -p temp-libbpf");
+  executeCommand("cd temp-libbpf");
+  executeCommand("git clone https://github.com/libbpf/libbpf");
+  executeCommand("git checkout "+ libbpf_version);
+  executeCommand("cd libbpf/src");
+  executeCommand("sudo make install_headers");
+  executeCommand("sudo make install_uapi_headers");
+  executeCommand("sudo prefix=/usr BUILD_STATIC_ONLY=y make install");
```

Comment:
`PREFIX` 
 
https://github.com/libbpf/libbpf/blob/20c0a9e3d7e7d4aeb283eae982543c9cacc29477/src/Makefile#L97

but `PREFIX` is already set to `/usr`